src/PostgREST/Types.hs

```diff
@@ -45,6 +49,7 @@ toMime CTTextCSV         = "text/csv"
 toMime CTTextPlain       = "text/plain"
 toMime CTOpenAPI         = "application/openapi+json"
 toMime CTSingularJSON    = "application/vnd.pgrst.object+json"
+toMime CTUrlEncoded      = "application/x-www-form-urlencoded"
```

Comment:
Code coverage drops because of this line. Is this ever going to be used? This would only be needed to **return** this mimetype, right? I doubt that will ever be the case.

BTW, that's the same case for the `CTAny` line below.

If our goal was to increase code coverage (not only technically but also for a nice number :D), this would be some simple things to do.

Reply:
Hmm.. yeah, it's unused.

How would I go about that? Just delete the lines? Not sure if the compiler would like that..

Reply:
Yeah, you're right. I didn't think that to the end. But for the code coverage stuff to be useful, we'll have to find a way to "remove" those false alarms, otherwise it's a pain to check for the important stuff.

It looks like the tool for the job are [hpc "overlays"](http://wiki.haskell.org/Haskell_program_coverage#Example). I will dig into this further once I find a bit of time and see what we can do.